lib/Slider/Slider.js

```diff
@@ -898,6 +898,7 @@ module.exports = kind(
 	*/
 	previous: function () {
 		this.set('value', this.value - this._jumpIncrementAmount);
+		if (this.popup && this.$.popup.getShowing()) return true;
```

Comment:
Hi @DavidUM, just spoke with @Djspaceg and Robert - let's make it consistent and display the popup when using the pointer to press the jump increment buttons.